packages/core/ios/LottieReactNative/ContainerView.swift

```diff
@@ -257,15 +261,15 @@ class ContainerView: RCTView {
         addSubview(next)
         animationView?.contentMode = contentMode
         animationView?.reactSetFrame(frame)
-        applyProperties()
+        animationView?.backgroundBehavior = .pauseAndRestore
+        animationView?.animationSpeed = speed
+        animationView?.loopMode = loop
+        applyColorProperties()
     }
-
-    func applyProperties() {
+    
+    func applyColorProperties() {
```

Comment:
Only apply color properties here. Setting animationSpeed or loopMode resets the animation